setup.sh

```diff
@@ -0,0 +1,295 @@
+fi
+logprint "Using web directory $WebFolder"
+
+service mailscanner stop
+
+if [ -d $WebFolder ]; then
+   read -p "Folder $WebFolder already exists. Content will get deleted. Do you want to continue?(y/n)[n]: " response
+   if [ -z $response ]; then
+       logprint "Stopping setup on user request"
+       exit
+   elif [ "$response" == "n" ]; then
+       logprint "Stopping setup on user request"
+       exit
+   fi
+   logprint "Clearing web directory"
+   rm -R $WebFolder
+fi
+
+logprint "Setting up sql credentials"
+#get sql credentials
+read -p "SQL user for mailwatch[mailwatch]:" SqlUser
+if [ -z $SqlUser ]; then
+    SqlUser="mailwatch"
+fi
+read -p "SQL password for mailwatch[mailwatch]:" SqlPwd
+if [ -z $SqlPwd ]; then
+    SqlPwd="mailwatch"
+fi
+read -p "SQL database for mailwatch[mailscanner]:" SqlDb
+if [ -z $SqlDb ]; then
+    SqlDb="mailscanner"
+fi
+read -p "SQL host of database[localhost]:" SqlHost
+if [ -z $SqlHost ]; then
+    SqlHost="localhost"
+fi
+logprint "Using sql credentials user: $SqlUser; password: $SqlPwd; db: $SqlDb; host: $SqlHost"
+
+if ! ( type "mysqld" > /dev/null 2>&1 ) ; then
+    read -p "No mysql server found. Do you want to install mariadb as sql server?(y/n)[y]: " response
+    if [ -z $response ] || [ $response == "y" ]; then
+        logprint "Start install of mariadb"
+        $PM install mariadb-server mariadb-client
+        mysqlInstalled="1"
+    else
+        mysqlInstalled="0"
+        logprint "Not installing mariadb."
+    fi
+else
+    mysqlInstalled="1"
+    logprint "Found installed mysql server and will use that"
+fi
+
+if [ "$mysqlInstalled" == "1" ]; then
+    read -p "Root sql user (with rights to create db)[root]:" SqlRoot
+    if [ -z $SqlRoot ]; then
+        SqlRoot="root"
+    fi
+    logprint "Creating sql database and setting permission. You now need to enter the password of the root sql user twice"
+    mysql -u $SqlRoot -p < "$InstallFilesFolder/create.sql"
+    mysql -u $SqlRoot -p --execute="GRANT ALL ON $SqlDb.* TO $SqlUser@localhost IDENTIFIED BY '$SqlPwd'; GRANT FILE ON *.* TO $SqlUser@localhost IDENTIFIED BY '$SqlPwd'; FLUSH PRIVILEGES"
+
+    read -p "Enter an admin user for the MailWatch web interface: " MWAdmin
+    read -p "Enter password for the admin: " MWAdminPwd
+    logprint "Create MailWatch web gui admin"
+    mysql -u $SqlUser -p$SqlPwd $SqlDb --execute="REPLACE INTO users SET username = '$MWAdmin', password = MD5('$MWAdminPwd'), fullname = 'Admin', type = 'A';"
+else
+    echo "You have to create the database yourself!"
+    EndNotice="$EndNotice \n * create the database, a sql user with access to the db and following properties user: $SqlUser; password: $SqlPwd; db: $SqlDb; host: $SqlHost"
+    EndNotice="$EndNotice \n * create an admin account for the web gui"
+fi
+
+#copy web files
+logprint "Moving MailWatch web files to new folder and setting permissions"
+mv "$InstallFilesFolder/mailscanner/" $WebFolder
+chown root:mtagroup $WebFolder/images
```

Comment:
need to check that `mtagroup` exists before we run the `chown` - it exists by default in centos 7.

Doesn't appear to exist by default in in debian 8, centos 6, Ubuntu 14.04 / 16.04